packages/flutter/lib/src/widgets/scroll_context.dart

```diff
@@ -61,6 +61,9 @@ abstract class ScrollContext {
   /// Set the [SemanticsAction]s that should be expose to the semantics tree.
   void setSemanticsActions(Set<SemanticsAction> actions);
 
+  /// Called when the content dimensions of the scroll position change.
+  void didChangeContentDimensions();
```

Comment:
Can we just add documentation for ScrollMetricsNotification here and vice versa - add documentation for this in ScrollMetricsNotification. I think having two separate APIs is ok, because they have very different uses. We may want to also say when they may be preferred, like if you want to listen for this change, choose the other one.